routes/ecom/modules/calculate-shipping.js

```diff
@@ -22,6 +22,19 @@ module.exports = appSdk => {
       return
     }
 
+    if (Array.isArray(config.services) && config.services.length) {
+      const { services } = config
+      const newShippingRules = shippingRules.map(rule => {
+        const foundService = services.find(service => service.service_code === rule.service_code)
+        if (foundService) {
+          ['free_shipping_all', 'product_ids'].forEach(prop => {
+            rule[prop] = foundService[prop]
+          })
+        }
+        return rule
+      })
+      shippingRules = newShippingRules
```

Comment:
Isso tudo era vontade de experimentar métodos "diferentes" de array prototype? 😄 

```suggestion
      shippingRules.forEach(rule => {
        const service = services.find(service => service.service_code === rule.service_code)
        if (service) {
          rule.free_shipping_all = service.free_shipping_all
          rule.product_ids = service.product_ids
        }
      })
```